config/environments/development.rb

```diff
@@ -76,6 +76,15 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
+  # To have the Rails log sent to the stdout and *not* a file,
+  # define RAILS_LOG_TO_STDOUT as anything.
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
+    logger           = ActiveSupport::Logger.new(STDOUT)
+    logger.formatter = config.log_formatter
+    config.logger    = ActiveSupport::TaggedLogging.new(logger)
+  end
```

Comment:
I think this is OK for now.  I wonder, however, if in the future we'd want to differentiate between `Rails` standard logs (`production.log`) and our app-specific logs (such as capturing rake task outcomes, user email events, etc.).

Perhaps we'll need a `APP_LOG_TO_STDOUT` ENV key in the future.

Reply:
This is the key that Rails 5 uses.  So perhaps we can have a different key for our stuff instead?
something like `ACTIVELOG_TO_STDOUT` ?

Would be easy to go ahead and put that in now.  Should I?